diff --git a/PC/pyconfig.h b/PC/pyconfig.h
index 40ced7d..42114aa 100644
--- a/PC/pyconfig.h
+++ b/PC/pyconfig.h
@@ -289,9 +289,7 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 			/* So MSVC users need not specify the .lib file in
 			their Makefile (other compilers are generally
 			taken care of by distutils.) */
-#			if defined(_DEBUG)
-#				pragma comment(lib,"python36_d.lib")
-#			elif defined(Py_LIMITED_API)
+#			if defined(Py_LIMITED_API)
 #				pragma comment(lib,"python3.lib")
 #			else
 #				pragma comment(lib,"python36.lib")
